actiontext/lib/action_text/attribute.rb

```diff
@@ -34,14 +34,11 @@ def #{name}=(body)
           end
         CODE
 
-        has_one :"rich_text_#{name}", -> { where(name: name) }, class_name: "ActionText::RichText", as: :record, inverse_of: :record, dependent: :destroy
+        has_one :"rich_text_#{name}", -> { where(name: name) },
+          class_name: "ActionText::RichText", as: :record, inverse_of: :record, autosave: true, dependent: :destroy
```

Comment:
Here’s the relevant bit: adding the `autosave: true` option to this association. This causes changes to the `ActionText::RichText`’s `body` attribute to be treated as autosaveable changes to the original record.